src/mixins/placeholder.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+/**
+ * CSS to style the selection psuedo-element.
+ *
+ * @example
+ * // Styles as object usage
+ * const styles = {
+ *   ...placeholder(styles)
+ * }
+ *
+ * // styled-components usage
+ * const div = styled.input`
+ *   ...placeholder(styles)
```

Comment:
just got back on this. I want to make sure I understand this situation, because it is more than just amending the types from what I can see. Here is what I'm seeing:

```javascript
const styles = {
  "background": "blue",
  "color": "red"
};

return css`${styles}`
```

This returns
```javascript
[
"background: blue; color: red;"
]
```

That means we would be passing `placeholder` an array with a single item that contains all the styles passed into it. Is that correct?

If that is the case I will need to refactor this, as spreading the passed in styles would not accommodate this use case.